backend/api/voting/forms.py

```diff
@@ -33,3 +36,43 @@ def save(self, commit=True):
     class Meta:
         model = Vote
         fields = ("submission", "value")
+
+
+class SendVotesForm(ContextAwareForm):
```

Comment:
Why we need two Votes form? If we want to save the votes altogether, ok, but can we use only one form, and pass only one item if we save a single vote?

Reply:
Yes, we do not need the two forms :) The plan is to batch update the votes instead of sending N requests.

Probably it's worth adding a reminder to delete the old mutation once the PR is complete

Reply:
We changed how the saving works and now we use the single `sendVote` only 🎉